### PR TITLE
ECO 557 Adding src/dst_network_id to Phantom GetEvent and GetEvents calls

### PR DESCRIPTION
**Pull request by bethwiron**

### CODE REVIEW

File: Apps/phironnet/ironnet.json [hunk 1]
```diff
@@ -1290,6 +1290,14 @@
                         "https://1.2.3.4/event?uuid=9f61e9e9af904540ac8bd730f5ed48d2"
                     ]
                 },
+                {
+                    "data_path": "action_result.data.*.events.*.src_network_id",
+                    "data_type": "string"
+                },
```
Comment (thao2) [inline Apps/phironnet/ironnet.json:1299]: Do we need to have "example_values" similar to the other fields?

Comment (bethwiron) [reply Apps/phironnet/ironnet.json:1299]: i think you're right i should add these--was looking at the examples before mike's new PR Comments PR which didn't always include examples --my bad. will add.

File: Apps/phironnet/ironnet.json [hunk 2]
```diff
@@ -1290,6 +1290,20 @@
                         "https://1.2.3.4/event?uuid=9f61e9e9af904540ac8bd730f5ed48d2"
                     ]
                 },
+                {
+                    "data_path": "action_result.data.*.events.*.src_network_id",
+                    "data_type": "string",
+                    "example_values": [
+                        "skynet-12345"
```
Comment (mjsell12) [inline Apps/phironnet/ironnet.json]: Lets make the sample network ID `sample-network`.

File: Apps/phironnet/ironnet.json [hunk 3]
```diff
@@ -1290,6 +1290,20 @@
...
+                    ]
+                },
+                {
+                    "data_path": "action_result.data.*.events.*.dst_network_id",
+                    "data_type": "string",
+                    "example_values": [
+                        "skynet-12345"
```
Comment (mjsell12) [inline Apps/phironnet/ironnet.json]: Lets make the sample network ID `sample-network`.

File: Apps/phironnet/ironnet.json [hunk 4]
```diff
@@ -1878,6 +1892,20 @@
                         "url"
                     ]
                 },
+                {
+                    "data_path": "action_result.data.*.event.src_network_id",
+                    "data_type": "string",
+                    "example_values": [
+                        "skynet-12345"
```
Comment (mjsell12) [inline Apps/phironnet/ironnet.json]: Lets make the sample network ID `sample-network`.

File: Apps/phironnet/ironnet.json [hunk 5]
```diff
@@ -1878,6 +1892,20 @@
...
+                    ]
+                },
+                {
+                    "data_path": "action_result.data.*.event.dst_network_id",
+                    "data_type": "string",
+                    "example_values": [
+                        "skynet-12345"
```
Comment (mjsell12) [inline Apps/phironnet/ironnet.json]: Lets make the sample network ID `sample-network`.

**Review by mjsell12 — CHANGES_REQUESTED** [review]

File: Apps/phironnet/ironnet.json (hunk 2 above)
Comment (bethwiron) [reply Apps/phironnet/ironnet.json]: fixed 👍

**Review by mjsell12 — APPROVED** [review]

**Pull request merged**